_inc/client/state/action-types.js

```diff
@@ -43,6 +43,12 @@ export const JETPACK_CONNECTION_STATUS_FETCH = 'JETPACK_CONNECTION_STATUS_FETCH'
 
 export const JETPACK_CONNECTION_TEST_FETCH = 'JETPACK_CONNECTION_TEST_FETCH';
 
+export const JETPACK_SITE_CONNECTED_PLUGINS_FETCH = 'JETPACK_SITE_CONNECTED_PLUGINS_FETCH';
+export const JETPACK_SITE_CONNECTED_PLUGINS_FETCH_RECEIVE =
+	'JETPACK_SITE_CONNECTED_PLUGINS_FETCH_SUCCESS';
+export const JETPACK_SITE_CONNECTED_PLUGINS_FETCH_FAIL =
+	'JETPACK_SITE_CONNECTED_PLUGINS_FETCH_FAIL';
```

Comment:
Were the multilines and tabs added by Prettier when you committed? It seems a bit out of style here. What do you think?

Reply:
I don't really like it either, but that wasn't me, and my local copy is still aligned in single lines.